### PR TITLE
fix: update storage class + fix duplicate name?

### PR DESCRIPTION
**Pull request by jamiecore**

### CODE REVIEW

File: kubernetes/k8_spec_ssd-cluster.yml [hunk 1]
```diff
@@ -1,13 +1,13 @@
 kind: PersistentVolumeClaim
 apiVersion: v1
 metadata:
-  name: eleuther-ssd-0
+  name: eleuther-ssd-cluster
```
Comment (StellaAthena) [inline kubernetes/k8_spec_ssd-cluster.yml:4]: `ssd-0` is how we have been labeling pod-independent permanent storage. `ssd-cluster` is the designation for the storage associated with the particular cluster that you are currently on.

Comment (jamiecore) [reply kubernetes/k8_spec_ssd-cluster.yml:4]: ok just noticed the same `metadata.name` in both the `k8_spec_ssd-cluster.yml` and `k8_spec_ssd-0.yml`

Comment (StellaAthena) [reply kubernetes/k8_spec_ssd-cluster.yml:4]: Hmmm. Looking through it, it does seem strange. But I just ran this a couple days ago without issue, and that makes me worried about changing it.

**Comment by sdtblck** [comment]
Hi @jamiecore - we're now using https://github.com/EleutherAI/kube for our deployment, the folder here will be deleted soon

**Pull request closed without merging**